jupyterlab/labapp.py

```diff
@@ -486,7 +486,11 @@ class LabApp(NotebookConfigShimMixin, LabServerApp):
     )
     flags["collaborative"] = (
         {"LabApp": {"collaborative": True}},
-        "Whether to enable collaborative mode.",
+        """To enable real-time collaboration, you must install the extension `jupyter_collaboration`.
+        This flag is now deprecated and will be removed in JupyterLab v5.
+        You can install it using pip for example:
+
+            python -m pip install jupyter_collaboration""",
```

Comment:
Would it read better if the order was switched around? Otherwise "it" seems to refer to the flag, not the extension.

```suggestion
        You can install it using pip for example:

            python -m pip install jupyter_collaboration
            
        This flag is now deprecated and will be removed in JupyterLab v5.""",
```

Reply:
You are right. I pushed a new commit with the change.

Thanks @krassowski